docs/customize/custom_fields/records.md

```diff
@@ -78,7 +78,7 @@ The keys of this config will be used to prefix field names. These prefixed names
 You can configure custom field by listing them in the `invenio.cfg`:
 
 ```python
-from invenio_vocabularies.services.custom_fields import TextCF, VocabularyCF
+from invenio_records_resources.services.custom_fields import TextCF, VocabularyCF
```

Comment:
I think there is bigger issue with this import. It should be checked across all docs and it should be 
```suggestion
from invenio_records_resources.services.custom_fields import TextCF
from invenio_vocabularies.services.custom_fields import VocabularyCF
```